issue_event_processing.ipynb

```diff
@@ -0,0 +1,458 @@
+    "\n",
+    "This notebook walks through multiple different models and algorithms for modeling the process. It is reccomended you start with a small number of event issues and gradually build multiple versions to have a deeper understanding of what events are changed the process model. At the bottom of the notebook is an examples section that shows what the models can generate. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Requirements\n",
+    "\n",
+    "-   Gihub Access Token \n",
+    "-   [Kaiaulu](https://github.com/sailuh/kaiaulu) R package to download data via CLI\n",
+    "-   Python environment with [pm4py](https://github.com/process-intelligence-solutions/pm4py) installed\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Python Imports"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 137,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas\n",
+    "import os\n",
+    "from datetime import datetime\n",
+    "import pm4py\n",
+    "import subprocess\n",
+    "\n",
+    "from pm4py.algo.discovery.dfg import algorithm as dfg_discovery\n",
+    "from pm4py.algo.discovery.heuristics import algorithm as heuristics_miner"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Download and Parse data with Kaiaulu ghevents.R (CLI)\n",
+    "\n",
+    "#### Set Up and Execute Download Command\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Save and set the current working directory to Kaiaulu\n",
+    "cwd = os.getcwd()\n",
+    "os.chdir(os.path.expanduser(\"~/Desktop/Kaiaulu/Working_issues/kaiaulu/\"))\n",
+    "\n",
+    "# To download use the download command specifing the <config_file> <github_token>\n",
+    "\n",
+    "command = f\"Rscript exec/ghevents.R download conf/kaiaulu.yml --token_path=~/.ssh/github_token\"\n",
+    "subprocess.run(command, shell=True, check=True)\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Execute Parse Command\n",
+    "\n",
+    "As stated above it is reccomended you start with only a few event issues. To do this you can open the created issue_output.csv with Excel or Google Sheets and modify the table to only include a few. \n",
```

Comment:
This is not useful. The user will not be able to create the files by hand with just this as instruction. Did you not say you created the fake generator for the examples? Where are they located? That's what they were for here.